src/excmds.ts

```diff
@@ -217,15 +217,14 @@ export async function guiset(rule: string, option: string) {
     // Could potentially fall back to sending minimal example to clipboard if native not installed
 
     // Check for native messenger and make sure we have a plausible profile directory
-    if (!await nativegate("0.1.1")) return
+    if (!await Native.nativegate("0.1.1")) return
     let profile_dir = ""
-    if (config.get("profiledir") === "auto") {
-        if (["linux", "openbsd", "mac"].includes((await browser.runtime.getPlatformInfo()).os)) profile_dir = await Native.getProfileDir()
-        else {
-            fillcmdline("Please set your profile directory (found on about:support) via `set profiledir [profile directory]`")
-            return
-        }
-    } else profile_dir = config.get("profiledir")
+    if (config.get("profiledir") === "auto" && ["linux", "openbsd", "mac"].includes((await browser.runtime.getPlatformInfo()).os)) profile_dir = await Native.getProfileDir()
```

Comment:
You need to wrap this in another try...catch as getProfileDir() will throw an error, so the nice friendly fillcmdline below will never run.